core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -433,7 +435,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
     this.consumers.head.commitSync(Map(tp -> new OffsetAndMetadata(5)).asJava)
   }
 
-  @Test(expected = classOf[TopicAuthorizationException])
+  @Test(expected = classOf[KafkaException])
```

Comment:
In this case, we don't have DESCRIBE on the topic so the broker returns UNKNOWN_TOPIC_OR_PARTITION and in OffsetCommitResponseHandler.handle() this error is not thrown as a KafkaException. Should we instead raise it as is ?